tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>As a non-time-evolution-expert, I would love to see a sentence here interpreting the graph, or with some sort of conclusion from this experiment.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>